Dockerfile.stable

```diff
@@ -1,82 +1,82 @@
-    /root/google-chrome-stable_current_x86_64.rpm
-RUN dnf install -y /root/google-chrome-stable_current_x86_64.rpm && \
-    rm -f /root/google-chrome-stable_current_x86_64.rpm
+RUN curl -LO https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm && \
+    dnf install -y google-chrome-stable_current_x86_64.rpm && \
+    rm -f google-chrome-stable_current_x86_64.rpm
 
 # chrome and chrome driver versions should match in order to avoid incompatibility
-
-# chromedriver
-ENV DESTFILE /root/chrome-driver/chromedriver_linux64.zip
-RUN CHROME_VERSION=$(rpm -q --qf "%{VERSION}\n" google-chrome-stable|sed -Ee 's/^(.*)\..*/\1/') && \
-    CHROME_DRIVER_VERSION=$(curl -s https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${CHROME_VERSION}) && \
-    mkdir -p /root/chrome-driver && \
-    curl -o ${DESTFILE} https://chromedriver.storage.googleapis.com/${CHROME_DRIVER_VERSION}/chromedriver_linux64.zip && \
-    unzip -d /root/chrome-driver/ ${DESTFILE} && \
-    rm -f ${DESTFILE}
-
-# selenium server
-ADD http://selenium-release.storage.googleapis.com/3.14/selenium-server-standalone-$SELENIUM_VERSION.jar \
-    /root/selenium-server/selenium-server-standalone.jar
+RUN CHROME_VERSION=$(rpm -q --qf "%{VERSION}\n" google-chrome-stable | sed -Ee 's/^(.*)\..*/\1/') && \
+    CHROME_DRIVER_VERSION=$(curl -s https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROME_VERSION) && \
+    curl -O https://chromedriver.storage.googleapis.com/$CHROME_DRIVER_VERSION/chromedriver_linux64.zip && \
+    unzip -d /usr/bin/ chromedriver_linux64.zip && \
+    chmod a+x /usr/bin/chromedriver && \
+    rm -f chromedriver_linux64.zip
 
 # firefox
-ADD https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 \
-    /root/firefox.tar.bz2
+RUN curl -LO https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 && \
+    tar -C . -xjvf firefox-$FIREFOX_VERSION.tar.bz2 && \
+    rm -f firefox-$FIREFOX_VERSION.tar.bz2 && \
+    firefox -headless -CreateProfile $DEFAULT_PROFILE_NAME && \
+    echo 'user_pref("app.update.enabled", false);' > $(find ~/.mozilla/firefox -name "*.$DEFAULT_PROFILE_NAME" -type d)/user.js
 
 # gecko for FF
-ADD https://github.com/mozilla/geckodriver/releases/download/$GECKODRIVER_VERSION/geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz \
-    /root/gecko.tar.gz
+RUN curl -LO https://github.com/mozilla/geckodriver/releases/download/$GECKODRIVER_VERSION/geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz && \
+    tar -C /usr/bin/ -xvf geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz && \
+    rm -f geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz
 
-#copying nginx config for selenium proxy
-ADD ./nginx-selenium.conf /etc/nginx/conf.d
+# selenium server
+ADD http://selenium-release.storage.googleapis.com/3.141/selenium-server-standalone-$SELENIUM_VERSION.jar $SELENIUM_PATH
```

Comment:
3.141 brings me to thought that we tie up our image to selenium 3.14. shouldn't this url be built based on global dockefile variable SELENIUM_VERSION ?